src/cmdlib.sh

```diff
@@ -164,6 +164,22 @@ disk_ignition_version() {
     fi
 }
 
+init_build_env() {
```

Comment:
Huge +1 to having these all in once place. Maybe we ought to comment them, or at least the tmp ones which are less obvious to the casual reader (e.g. why have `manifest` and `manifest_tmp_json`).